### PR TITLE
[#2636] Migrate stealth disadvantage on armor to properties

### PR DESCRIPTION
**Pull request by arbron**
The "Imposes Stealth Disadvantage" option on equipment seems like the only other property that should reasonable migrated into the new item properties.

### CODE REVIEW

File: module/applications/item/item-sheet.mjs [hunk 1]
```diff
@@ -139,7 +139,7 @@ export default class ItemSheet5e extends ItemSheet {
       const isAmmo = item.system.type?.value === "ammo";
       const valids = CONFIG.DND5E.validProperties[item.type];
       context.properties = Object.entries(CONFIG.DND5E.itemProperties).reduce((obj, [k, v]) => {
-        if ( valids.has(k) || (isAmmo && v.isPhysical) ) {
+        if ( valids.has(k) || (isAmmo && v.isPhysical) || (item.system.isArmor && k === "stealthDisadvantage") ) {
```
Comment (krbz999) [inline module/applications/item/item-sheet.mjs:142]: This single line of code is gonna explode in length over time, isn't it.

Comment (Fyorl) [inline module/applications/item/item-sheet.mjs:142]:
```suggestion
        if ( valids.has(k) || (isAmmo && v.isPhysical) || (item.system.isArmor && (k === "stealthDisadvantage")) ) {
```

File: module/data/item/container.mjs [hunk 2]
```diff
@@ -61,7 +60,7 @@ export default class ContainerData extends SystemDataModel.mixin(
     Object.defineProperty(this.capacity, "weightless", {
       get() {
         foundry.utils.logCompatibilityWarning(
-          "The `system.capacity.weightless` value on containers has migrated into a property.",
+          "The `system.capacity.weightless` value on containers has migrated to the 'weightlessContents' a property.",
```
Comment (Fyorl) [inline module/data/item/container.mjs:63]:
```suggestion
          "The `system.capacity.weightless` value on containers has migrated to the 'weightlessContents' property.",
```

File: module/data/item/equipment.mjs [hunk 3]
```diff
@@ -174,4 +184,18 @@ export default class EquipmentData extends SystemDataModel.mixin(
...
+
+  /* -------------------------------------------- */
+
+  /**
+   * Does this armor impose disadvantage on stealth checks?
+   * @type {boolean}
+   */
+  get stealth() {
+    foundry.utils.logCompatibilityWarning(
+      "The `system.stealth` value on equipment has migrated to the 'stealthDisadvantage' a property.",
```
Comment (Fyorl) [inline module/data/item/equipment.mjs:196]:
```suggestion
      "The `system.stealth` value on equipment has migrated to the 'stealthDisadvantage' property.",
```

**Review by Fyorl — CHANGES_REQUESTED** [review]

File: module/applications/item/item-sheet.mjs (hunk 1 above)
Comment (arbron) [reply module/applications/item/item-sheet.mjs:142]: I have another PR to fix that :)

**Review by Fyorl — APPROVED** [review]

**Pull request merged**